### PR TITLE
Add p-notifications from vanilla-framework

### LINKED ISSUE #28: Add notifications
Currently admonitions in docs have no style, as outlined in [this issue](https://github.com/ubuntudesign/vanilla-docs-theme-design/issues/3).

The style should follow the same style as inline [Vanilla notifications](https://docs.vanillaframework.io/en/patterns/notification) including icons.

Steps:
- Add admonitions to documentation builder
- Include notifications CSS in docs theme

### PR DESCRIPTION
**Pull request by nottrobin**
We need to support notifications for the docs.

I've created `docs-p-notifications` which currently just includes
`vf-p-notification` directly, but could be extended in future if we have
different requirements for the docs theme.

Fixes #28.

QA
--

``` bash
npm install
node_modules/.bin/gulp build
```

Now open `demo/index.html` in your browser, and see notifications working at the bottom.

### CODE REVIEW

File: scss/_patterns_notifications.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+@import 'vanilla-framework/scss/global_functions';
+@import 'vanilla-framework/scss/patterns_notifications';
+
+@mixin docs-p-notifications {
```
Comment (anthonydillon) [inline scss/_patterns_notifications.scss]: This should be `theme-p-notifications` to follow convention.

Comment (nottrobin) [inline scss/_patterns_notifications.scss]: ORLY? What if (and I know we don't want to encourage this) you needed to mix and match mixins from two different themes? I've had to do it in the past. Surely namespacing should be as specific as possible exactly to avoid this sort of conflict?

Comment (nottrobin) [inline scss/_patterns_notifications.scss]: In any case, every pattern mixin in this codebase is called `docs-p-*`, so I imagine if we're to update it it should be across the whole codebase and so should probably be a separate PR.

Comment (anthonydillon) [inline scss/_patterns_notifications.scss]: Ok

**Comment by yaili** [comment]
Should this not have been design reviewed in develop before being committed to master? @barrymcgee @nottrobin @anthonydillon ?

**Comment by barrymcgee** [comment]
Yes, it should have but this PR was created before I created the develop branch and made it default (earlier today)... in future, all PRs will propose to develop by default.

**Comment by yaili** [comment]
@barrymcgee ok, so where can it be reviewed?

**Comment by nottrobin** [comment]
@yaili I did think that because this is simply a plain implementation of the [existing notifications](https://docs.vanillaframework.io/en/patterns/notification), with no design changes at all, it didn't need design review.

**Comment by yaili** [comment]
@nottrobin no worries. I think the process is that everything is merged onto develop once it's been code reviewed, and it's design reviewed in develop before release. Right, @barrymcgee @anthonydillon ?

**Comment by nottrobin** [comment]
@yaili you can review it here, for example: https://docs.staging.ubuntu.com/maas/2.1/en/

**Comment by nottrobin** [comment]
@yaili yep, I was aware of that for vanilla-framework. As @barrymcgee said, up until very recently vanilla-docs-theme didn't have a develop branch. I'll make sure I merge into there in future.

**Comment by yaili** [comment]
Cool, thanks! 👍
